maintainers/maintainer-list.nix

```diff
@@ -2673,6 +2673,11 @@
     github = "lejonet";
     name = "Daniel Kuehn";
   };
+  luis = {
```

Comment:
Looks like this commit is no longer needed as you've already been added to the maintainer list :tada: